draft-ietf-core-multipart-ct.md

```diff
@@ -166,13 +166,13 @@ or any residual data left after processing the CBOR data item.
 (This generally means the representation is not processed at
 all except if some streaming processing has already happened.)
 
-# Usage Example: Observing Resources
+# Usage Example: CoAP Observing Resources
```

Comment:
typo:  s/CoAP Observing Resources/Observing CoAP Resources/

Reply:
I think we need to keep in mind that this document is far advanced in the IETF process.
This is not really the time for Brownian motion...

Reply:
> typo:  s/CoAP Observing Resources/Observing CoAP Resources/

Will fix.

Reply:
> I think we need to keep in mind that this document is far advanced in the IETF process.
> This is not really the time for Brownian motion...

Much bigger changes than this have been done in the last second :)

If there is anyway going to be a new version of the draft this change can be done.

Reply:
A lot of things have been done; this doesn't mean it's right.

Reply:
I fail to see what's added here.

Reply:
The text talks about response codes (2.05, 2.03 etc), Observe options etc, without any reference to what they are. Keep in mind that this is not a CoAP specific document, so one should not have to assume that the text talks about CoAP.

Reply:
There is a reference to RFC 7641 right in the first paragraph.
(It's not like CoAP hasn't been mentioned before in this document.
It's even in the title!)

Reply:
I know what the title says, but I was told the mechanism is generic. Also, unless I've missed it, CoAP is only used as an example ("such as") in the Abstract and Introduction, and later in the Example section. Everything else is about the MIME type and the encoding, isn't it?

So, either the draft needs to be scoped to CoAP, in which case it should be indicated in the Abstract and Introduction, OR it should be made generic - in which case the title should probably be changed).

Reply:
The media type is generic.  The example is about "observing resources", which as of today is CoAP-specific.  There are no other examples (except the trivial ones in the exposition), because they are, err, trivial.

Reply:
> The media type is generic.  The example is about "observing resources", which as of today is 
> CoAP-specific. There are no other examples (except the trivial ones in the exposition), <
> because they are, err, trivial.

What is "today" may not be "tomorrow". One shall assume that information that the reader might take for granted today will be the same also in the future.

However, if you want to keep it generic, but still scope this document to CoAP, you could add some text, saying something like:

"While the MIME type defined in this document is generic, the document was written with CoAP in mind. Other specifications might define how to use the MIME type with other protocols"

That is quite typical in other specifications, where the WG does not want to prevent the defined mechanism to be used elsewhere in future, but does not want to deal with the details of that within the current specifications.